stl/inc/format

```diff
@@ -1848,6 +1877,90 @@ struct formatter {
     formatter operator=(const formatter&) = delete;
 };
 
+template <class _Ty, class _CharT, _Basic_format_arg_type _ArgType>
+struct _Formatter_base {
+
+    using _Pc = basic_format_parse_context<_CharT>;
+
+    typename _Pc::iterator parse(_Pc& _ParseCtx) {
+        _Specs_checker<_Dynamic_specs_handler<_Pc>> _Handler(_Dynamic_specs_handler<_Pc>(_Specs, _ParseCtx), _ArgType);
+        auto _It = _ParseCtx.begin();
+        _It._Seek_to(_Parse_format_specs(_ParseCtx.begin()._Unwrapped(), _ParseCtx.end()._Unwrapped(), _Handler));

```

Comment:
Urgh, I really do not like this shenanigance.

Reply:
I don't either, I'm not sure it was a good idea to use char* instead of keeping iterators all the way through. It turns out libfmt uses it's own string_view implementation that always has char* iterators, and so it skips this